### PR TITLE
Minor fixes in security chapter

### PR DESCRIPTION
**Pull request by vanmaegima**

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 3b1231e406bc6e3ee1d874cbd7de7b728314db82 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1050/docs/artifacts/html/index.html

**Review by angolini — APPROVED** [review]
LGTM

File: source/reference-manual/security/secure-boot-apalis-imx6-sec.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+.. warning::
+
+    It is recommended to read :ref:`ref-secure-boot-imx` before proceeding with
+    the following steps.
+
+How to enable
+-------------
+
+In the ``ci-scripts`` git repository from the FoundriesFactory, update the
```
Comment (munoz0raul) [inline source/reference-manual/security/secure-boot-apalis-imx6-sec.rst:23]: Customers are not allowed to change the ci-scripts. We have to discuss internally what would be the best way to allow customers to edit the Machine.

Comment (MrCry0) [reply source/reference-manual/security/secure-boot-apalis-imx6-sec.rst:23]: @munoz0raul @angolini if customers should ask support to do any changes in ci-scripts, I think, the following pages need such warning as well:
https://docs.foundries.io/latest/reference-manual/factory/factory-definition.html
https://docs.foundries.io/latest/reference-manual/ota/advanced-tagging.html
https://docs.foundries.io/latest/reference-manual/linux/linux-oss-compliance.html?highlight=mfgtools#how-to-remove-packages-under-gplv3-family-license

**Comment by doanac** [comment]
Docs for 3d829698a5c204f8fde49278254e71f2d52874bb are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1058/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for bee2d7391354dc98d96a31942875b44762bbf545 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1059/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for bee2d7391354dc98d96a31942875b44762bbf545 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1060/docs/artifacts/html/index.html

File: source/reference-manual/factory/factory-definition.rst [hunk 2]
```diff
@@ -104,6 +104,12 @@ lmp:
 
       **Default:** Set by user during :ref:`gs-signup`
 
+ .. note::
+
+     Subscribers are not allowed to change the factory machine by themselves for
+     product reasons. If needed, ask a support engineer to update the machine
+     definition for your FoundriesFactory.
```
Comment (angolini) [inline source/reference-manual/factory/factory-definition.rst:111]: Maybe we can explain that there is a limit on computation available for each factory and that's why the machines parameter is limited. Maybe something like:

"The CI is configured to not accept changes in the ``machines:`` parameter due to computation limits. If needed, as a upport engineer to update the machine definition for your FoundriesFactory."

Do we want to add the error from the CI, so people searching the error might find it here?

**Comment by doanac** [comment]
Docs for db7a792cb600214c3885fa01b2a48afb9ee4b016 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1062/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 51ce066861e7ea39f42e2bfd6ce272e8de166885 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1063/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for e7403bcabea3f3311be68ea438a48f9cb6f0735d are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1066/docs/artifacts/html/index.html

**Pull request merged**